### DISCUSSION TITLE
绕过Zepp应用给手表安装应用

### DISCUSSION THREAD

**Discussion by OuNiSang** [zepp-health/zeppos-forum#45] [User Feedback]
我在国外买了Trex 2， 想要安装一个Google Map的提醒应用在手表上。因为之前用小米手环在Zepp Life记录了很多信息所以就想沿用国内小米账号。但是国内账号好像无法安装 Notify G Map 或者 Navigation Wear 这些应用。所以想请问下有没有方法可以绕过Zepp给手表直接安装应用的方法。

**Comment by htoooth** [comment]
Hi, @OuNiSang
sorry, there is no way to install app without zepp app.  You should register a American account if you want to use these app.

**Reply by OuNiSang** [reply]
Hi, thank you for your reply! 

Is there any way I may transfer the personal data from the mainland account to the American account? I do can export the data from the mainland account but have now idea how to import it. Is there a third-party solution to this? 

Thanks!

**Reply by htoooth** [reply]
Hi, @OuNiSang 
I have consulted with my colleagues. I'm sorry that there is no way to migrate personal data from one account to another. There is also no way to modify the account location.

**Reply by OuNiSang** [reply]
Hi Htooth

Thank you for reply!  It okay, I switched to an US account now, thanks for replying to my questions!
